### ISSUE TITLE
Mend CLI

### ISSUE THREAD

**Issue by aairey** [trimble-oss/gh-actions#16]
We should start using Mend CLI vs the WSS Unified Agent.

https://docs.mend.io/bundle/wsk/page/migrating_to_the_mend_cli_from_the_unified_agent.html

Wanting to contribute a new action, or should I adapt the existing one?

Also, side-note, this actions are not yet (automatically) versioned.

**Comment by coliff** [comment]
Hey thanks for reporting! I'd be happy for you to adapt the existing one or create a new one - either is fine! Thanks!

**Comment by aairey** [comment]
Yeah not sure if we need backwards compatibility.

I would really suggest to version these actions though. As most are
pointing to @main most likely, we are not able to make breaking changes
without impact.
Kind Regards,

Andy Airey

**Comment by coliff** [comment]
> I would really suggest to version these actions though

Yes, I agree! I'm keen to setup Mend using the correct way on a few repositories so please make a PR when you can. thanks!

**Comment by cyberdef** [comment]
We should sync on the new CLI. I'm away for a few days at a conference, but yes, we are moving solely to the new Mend CLI (for multiple reasons - avoids being out of date, has multiple functions including SAST capability, which we are now fully licensed for).

**Comment by coliff** [comment]
Heya @cyberdef @aairey - it's been a while since we discussed this, but maybe it makes sense to delete this repo altogether.

Mend should only be used via the GitHub app I believe and the only other GitHub Action in this repo is a Google Notifications one which didn't work too well and wasn't helpful. Let me know what you think...
